README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
+1. Press the space bar to begin.
+2. Use the up and down arrow keys to rotate the shape.
+3. Use the left and right arrow keys to position the shape.
```

Comment:
The problem here is that too many Tetris pieces are similar in color. In your review, try to use contrasting colors to make the pieces stand out from one another to make it easier to play the game.